students/eowyn/session06/mailroom/test_mailroom.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+
+import mailroom
+import pytest
+import os.path
+
+
+def test_safeinput():
```

Comment:
do you really want this to return None?

again -- it's OK not to test user-interaction!